packages/core/src/App/Containers/Modals/mt5-notification/mt5-notification.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { connect } from 'Stores/connect';
+import PropTypes from 'prop-types';
+import { Modal, DesktopWrapper, MobileDialog, MobileWrapper, UILoader, Button } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import MT5NotificationDescription from './mt5-notification-description';
+
+const MT5Notification = ({ setMT5NotificationModal, is_mt5_notificaiton_modal_visible, disableApp, enableApp }) => {
+    const onclickFunc = () => {
+        if (is_mt5_notificaiton_modal_visible) {
+            setMT5NotificationModal(false);
+        } else {
+            setMT5NotificationModal(true);
+        }
+    };
```

Comment:
```suggestion
    const clickHandler = () => {
        setNotificationModal(!is_mt5_notificaiton_modal_visible)
    };
```
nit: u can simplify this right